Add new aggregation (min, max, sum, value_count) description to docs (#2738)
quickwit-oss/quickwit: docs/reference/aggregation.md

Example request, histogram with stats in each bucket:

#### Aggregating on datetime fields

> The `datetime` fields are handled the same way as other numric fields. However, all values in the requests such is intervals, offsets, bounds, and range boundaries need to be expressed in microseconds.

Reviewer: ```suggestion
Fields of type `datetime` are handled the same way as any numeric field. However, all values in the requests such as intervals, offsets, bounds, and range boundaries need to be expressed in microseconds.
```

}
```

### Count

> A single-value metric aggregation that computesthat counts the number of values that are extracted from the aggregated documents.

Reviewer: ```suggestion
A single-value metric aggregation that counts the number of values that are extracted from the aggregated documents.
```
